test/unit/geojson.js

```diff
@@ -17,12 +17,12 @@ function parse(geojson) {
 describe('GeoJsonParser', function () {
     it('should set all z coordinates to 1', () =>
         parse(holes).then((collection) => {
-            assert.ok(collection.features[0].vertices.every((v, i) => i == 0 || ((i + 1) % 3) != 0 || v == 0));
+            assert.ok(collection.features[0].vertices.every((v, i) => i == 0 || ((i + 1) % 3) != 0 || (v + collection.position.z) == 0));
```

Comment:
The condition `i == 0` could be removed, since when this condition is true, the second condition (`(i + 1) % 3 != 0`) is always false.

The same goes for the line 25.